tools/Google.Cloud.Tools.ReleaseManager/UpdateProjectPropertiesCommand.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"):
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Tools.Common;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Google.Cloud.Tools.ReleaseManager;
+
+public sealed class UpdateProjectPropertiesCommand : CommandBase
+{
+    public UpdateProjectPropertiesCommand()
```

Comment:
This is to run whenever we delete a "special" project then right?

Reply:
Maybe we can have a github action run this on submit, or a check based on this. Similar to how we check "generate projects".